webpack.config.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+
+module.exports = {
+    entry: './src/index.js',
```

Comment:
```suggestion
    entry: './index.js',
```
так как у вас данный файл не в `src`, а в корневой папке